assignments/objects.js

```diff
@@ -18,20 +18,60 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const intern1 = {
```

Comment:
It would be easier to grab the data from each intern object if the object name were the same as the intern name in this context.